webpack/react_app/components/FeaturesDropdown/index.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  DropdownItem,
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleAction,
+} from '@patternfly/react-core';
+import { push } from 'connected-react-router';
+
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { STATUS } from 'foremanReact/constants';
+
+import { REX_FEATURES_API, NEW_JOB_PAGE } from './constant';
+import { runFeature } from './actions';
+
+const FeaturesDropdown = ({ hostId }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const {
+    response: { results: features },
+    status,
+  } = useAPI('get', foremanUrl(REX_FEATURES_API));
+  const dispatch = useDispatch();
+  const dropdownItems = features
+    ?.filter(feature => feature.host_action_button)
+    ?.map(({ name, label, id, description }) => (
+      <DropdownItem
+        onClick={() => dispatch(runFeature(hostId, label, name))}
+        key={id}
+        description={description}
+      >
+        {name}
+      </DropdownItem>
+    ));
+  const scheduleJob = [
+    <DropdownToggleAction
+      onClick={() => dispatch(push(`${NEW_JOB_PAGE}=${hostId}`))}
+      key="schedule-job-action"
+    >
+      {__('Schedule a job')}
+    </DropdownToggleAction>,
+  ];
+
+  return (
+    <Dropdown
+      alignments={{ default: 'right' }}
+      onSelect={() => setIsOpen(false)}
+      toggle={
+        <DropdownToggle
+          splitButtonItems={scheduleJob}
+          toggleVariant="secondary"
```

Comment:
looks like it doesn't work... no `secondary` nor `primary`

Reply:
I've noticed this too. Maybe we should file a PF issue..

Reply:
I made [this Codepen](https://codesandbox.io/s/holy-breeze-70rwf?file=/index.js) intending to file an issue, but `toggleVariant` seems to work fine there...

Reply:
So...the `toggleVariant` was added in `@patternfly/react-core@4.177.0`
currently we use `"@patternfly/react-core": "~4.162.2"`
:\

anyway, it's not a bug and should be resolved as soon as we will update pf

Reply:
But the old prop `isPrimary` doesn't seem to do anything either :thinking: